lib/SyTest/Homeserver.pm

```diff
@@ -173,25 +173,6 @@ sub configure
    $self->SUPER::configure( %params );
 }
 
-=head1 METHODS
-
-=head2 kill_and_await_finish
-
-   $hs->kill_and_await_finish
-
-Kill any processes started for the homeserver, and wait for them to exit.
-
-Returns a Future when the proceses have exited.
-
-It is expected that this will be overridden by subclasses.
-
-=cut
-
-sub kill_and_await_finish
```

Comment:
this is now provided by SyTest::Homeserver::ProcessManager, so doesn't need a dummy implementation here.